src/components/FlawComments/FlawComments.vue

```diff
@@ -46,28 +43,33 @@ const commentLabels = computed(() => {
     .map(key => `${key} Comments`);
 });
 
-const commentTooltips: Record<CommentType, string> = {
-  [CommentType.Public]: 'Bugzilla Public - This comments are visible to everyone.',
-  [CommentType.Private]: 'Bugzilla Private - This comments are visible to Red Hat associates.',
-  [CommentType.Internal]: 'Jira Internal - This comments are visible to team members with required permissions.',
-  [CommentType.System]: 'Bugzilla System - This are auto-generated private comments.',
-};
-
 const selectedTab = ref(CommentType.Public);
 const handleTabChange = (index: number) => {
   selectedTab.value = index;
 };
 
+const unifiedComments = computed(
+  () => {
+    const unifiedCommentsArray = [
+      ...props.publicComments,
+      ...props.privateComments,
+      ...props.systemComments,
+      ...props.internalComments,
+    ];
+    return orderCommentsByDate(unifiedCommentsArray);
+  },
+);
+
 const displayedComments = computed(() => {
   switch (selectedTab.value) {
     case CommentType.Public:
-      return props.publicComments;
+      return orderCommentsByDate(props.publicComments);
     case CommentType.Private:
-      return props.privateComments;
+      return orderCommentsByDate(props.privateComments);
     case CommentType.Internal:
-      return props.internalComments;
+      return orderCommentsByDate(props.internalComments);
     case CommentType.System:
-      return props.systemComments;
+      return orderCommentsByDate(props.systemComments);
     default:
       return [];
   }
```

Comment:
If we sort the comments directly in `useFlawComments` we could have a map of `CommentType` to comments array.
```typescript
const commentsByType = {
  [CommnetType.Pulblic]: publicComments,
  [CommnetType.Private]: privateComments,
  [CommnetType.Internal]: internalComments,
  [CommnetType.System]: systemComments,
}
```

using the comments then is just:
```vue
<CommentList :commentList="commentsByType[selectedTab]" /> 
```

This way we can remove the computed property, I don't think there's any performance difference (there's some, as is one computed ref less, and direct access to an in memory dictionary), and this `commentsByType` map could be part of the `useFlawComments` composable as well